CONTRIBUTING.md

```diff
@@ -63,14 +60,20 @@ reporting a bug, make sure the issue has not already been reported.
 
 #### Suggest an Enhancement
 
-To suggest an enhancement, create an issue on GitHub with the label `proposal` using the available feature issue
-template.
+To suggest an enhancement, [open an idea][idea] on GitHub discussions. We highly recommend that you open a discussion
```

Comment:
@mpstefan would it help if I added a line about opening an enhancement request issue here? It's not explicitly mentioned, which may be what's causing the confusion.